lib/mp4/transmuxer.js

```diff
@@ -140,38 +213,45 @@ VideoSegmentStream = function(track) {
 
   delete track.minPTS;
 
-  this.push = function(data) {
-    collectDtsInfo(track, data);
+  this.gopCache = [];
```

Comment:
It's used (well, cleared) by the Transmuxer function so I don't consider it private.